src/pages/iou/IOUCurrencySelection.js

```diff
@@ -84,7 +84,7 @@ function IOUCurrencySelection(props) {
             if (_.isEmpty(backTo) || props.navigation.getState().routes.length === 1) {
                 Navigation.goBack();
             } else {
-                Navigation.navigate(`${props.route.params.backTo}?currency=${option.currencyCode}`);
+                Navigation.goBack(`${props.route.params.backTo}?currency=${option.currencyCode}`, true);
```

Comment:
NAB: I did this, idk if it helps at all. I thought it did at one point. We already have an issue to fix the related bug anyways.

Reply:
@neil-marcellini Do you have any particular reason for this change?
This change create a Deploy Blocker